docs/api/scheduler.md

```diff
@@ -271,6 +271,28 @@ For the default queue hierarchy (only `root.default` leaf queue exists) a simila
 
 ## Applications
 
+### Partition applications
+
+Fetch all Applications for the given Partition/State combination and displays general information about the applications like used resources, queue name, submission time and allocations.
+Only following application states are allowed: active, rejected, completed. Active is a fake state that represents all application states except completed and rejected.
+For active state, can narrow the result by status query parameters(case-insensitive). For example, can fetch `Running` applications for the default partition by
+`/ws/v1/partition/default/applications/active?status=running`.
+
+**URL** : `/ws/v1/partition/:partition/applications/:state`
```

Comment:
this comment is unrelated to this PR.

Should we support `/ws/v1/partition/{partitionName}/queue/{queueName}/applications/:state`? that makes our APIs more consistent since both `/ws/v1/partition/{partitionName}/application/{appId}` and `/ws/v1/partition/{partitionName}/queue/{queueName}/application/{appId}` are available now.

Reply:
I have opened https://issues.apache.org/jira/browse/YUNIKORN-2461 for that